src/components/geolocation/GeolocationDirective.js

```diff
@@ -109,9 +109,11 @@ goog.require('ga_throttle_service');
         // Get heading depending on devices
         var headingFromDevices = function() {
           var hdg = deviceOrientation.getHeading();
-          var orientation = window.orientation;
+          var orientation = $window.orientation ||
+              $window.screen.orientation.angle;
```

Comment:
This will trigger an error on IE. You ned to test if `$window.screen.orientation` exists first